login-with-be-magiclink/backend/index.js

```diff
@@ -0,0 +1,123 @@
+import express from 'express';
+import fetch from 'node-fetch';
+import { common } from '@ciam-expressjs-vanilla-samples/shared';
+import * as querystring from 'querystring';
```

Comment:
```suggestion
import querystring from 'querystring';
```
if not working, add types